github/ipfs.yml

```
@@ -3602,6 +3601,23 @@ repositories:
         - Repos - JavaScript
         - admin
     visibility: public
+  js-kubo-rpc-client:
+    collaborators:
+      admin:
+        - SgtPooki
+      push:
+        - web3-bot
+    description: A client library for the Kubo RPC API
+    teams:
+      admin: []
+      pull: []
+      push: []
+    topics:
+      - http-client
+      - ipfs
+      - kubo
+      - rpc-client
+    visibility: public
```

Bringing back settings that weren't properly applied before. 
```suggestion
    branch_protection:
      master: {}
      release: {}
      release-*: {}
    collaborators:
      push:
        - web3-bot
    description: A client library for the Kubo RPC API
    teams:
      admin:
        - admin
        - w3dt-stewards
        - ipdx
      pull:
        - contributors
      push:
        - Maintainers
        - Merge - Go
        - Repos - Go
    topics:
      - ipfs
      - http-client
      - rpc-client
      - kubo
    visibility: public
```